src/client++/TurnMsgLib.h

```diff
@@ -75,7 +75,7 @@ class StunAttrIterator {
 	/**
 	 * Iterator constructor: creates iterator on raw messagebuffer.
 	 */
-	StunAttrIterator(uint8_t *buf, size_t sz) throw (WrongStunBufferFormatException) :
+	StunAttrIterator(uint8_t *buf, size_t sz) :
```

Comment:
Removing exceptions may be not right?
http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2016/p0003r5.html#4.0
See: Compatibility Concerns

Missing try catch and throw exception..

Reply:
exception classes should derive  from the std:exception
>     #include <stdexcept>
>     class MyException : public std::runtime_error {
>     public:
>       MyException() : std::runtime_error("MyException") { }
>     };
>     void f()
>     {
>        // ...
>        throw MyException();
>     }

Reply:
thank both of you, i'll fix it.